breol_base/breol_base.module

```diff
@@ -37,7 +37,7 @@ function breol_base_form_alter(&$form, &$form_state, $form_id) {
     if (module_exists('ding_adgangsplatformen')) {
       $form['adgangsplatformen-wrapper'] = array(
         '#type' => 'fieldset',
-        '#description' => t('Log in via adgangsplatformen'),
+        '#description' => t('Log in adgangsplatformen'),
```

Comment:
Are you changing the text to force a new translation? I'm not sure this wording makes sense; I think “Log in with Adgangsplatformen” or (“Sign in …”) is better.

Reply:
Fixed